oauth-client/src/main/java/io/strimzi/kafka/oauth/client/JaasClientOauthLoginCallbackHandler.java

```diff
@@ -66,7 +68,7 @@ public void configure(Map<String, ?> configs, String saslMechanism, List<AppConf
             String endpoint = config.getValue(ClientConfig.OAUTH_TOKEN_ENDPOINT_URI);
 
             if (endpoint == null) {
-                throw new RuntimeException("Access Token not specified ('oauth.access.token'). OAuth2 Token Endpoint ('oauth.token.endpoint.uri') should then be set.");
+                throw new RuntimeException("Access Token not specified (OAUTH_ACCESS_TOKEN). OAuth2 Token Endpoint (OAUTH_TOKEN_ENDPOINT_URI) should then be set.");
```

Comment:
What is the reason for changing these? the configuration options are still valid, or?

Reply:
It's for consistency reasons, because in other places it's also env var name that's used rather than system property name. It does not affect logic - options stay the same and are still valid.

Reply:
I think thsi could be a bit confusing because on the server side we use the options and not env. vars. But you are right - it is that way in most other messages so lets keep it consitent.